xlab-classes/cse442-spring2022-team-bug-busters
# Added in functionality for the DB regarding point system, added in 3 …

…different leaderboards to display to the users in the frontend.

**Task Tests**

*Test 1*
1. Start the server with 'php -S localhost:8080 -t ./api'
2. Go to http://localhost:8080/
3. Verify that the wins leaderboard appears with the table organized from most wins to least wins.
4. Verify that the loses leaderboard appears with the least losses to the most losses.
5. Verify that the points leaderboard appears with the most points to the lowest points.
